config.json

```diff
@@ -2,44 +2,44 @@
 "general": {},
 
 "graphics": {
-    "resolution_x": 2560,
-    "resolution_y": 1600,
+    "resolution_x": 1920,
+    "resolution_y": 1080,
```

Comment:
yes we should have a smaller screen by default; even 1080x800 might be better for a default

Reply:
How about 1280x960? That should work almost everywhere, while giving us a little more screen real estate than we've had in the past by default.

Reply:
that should be fine. Just so long as its something that fits well in a window.

Reply:
@royfalk what are your thoughts on this?

Reply:
I'm not Roy, of course, but I do have some thoughts. 
Firstly, 1920x1080 seems to be a sane default. 
It's not related to the travel mode multiplier though, so maybe have a new pr for this?

Looking down the road we would like to have this configure-able somehow, and surviving game updates, so there might be some things to discuss in the PR.

Reply:
Agree with Evert.
Full HD has been a standard for a decade now.
Also on the bit about a new  PR and persistence of the settings.

Reply:
@evertvorster @royfalk Good suggestions. I have split the resolution change off into its own PR: #170

Reply:
And the new Issue is: https://github.com/vegastrike/Assets-Masters/issues/68

Reply:
For reference, my laptop display is 1920x1080 to start with.
We should have a sane default for folks with a single monitor that can fit in a window on that monitor.

Reply:
> For reference, my laptop display is 1920x1080 to start with. We should have a sane default for folks with a single monitor that can fit in a window on that monitor.

Sounds reasonable. My laptop display is that same resolution, FWIW.

Reply:
Thanks - nice that #170 split out just the resolution change